Resources/config/symfony_services.yml

```diff
@@ -50,7 +50,7 @@ services:
     auto1.api.serializer.normalizer.datetime:
         class: Symfony\Component\Serializer\Normalizer\DateTimeNormalizer
         arguments:
-            - '%auto1.api.serializer.normalizer.datetime.format%'
+            - ['%auto1.api.serializer.normalizer.datetime.format%']
```

Comment:
looks like it should be something like this:
`[{datetime_format: '%auto1.api.serializer.normalizer.datetime.format%'}]`

See https://github.com/symfony/symfony/blob/4.2/src/Symfony/Component/Serializer/Normalizer/DateTimeNormalizer.php#L39-L53